packages/ui-svelte/src/Vercel.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import clsx from 'clsx'
+  export let className = ''
+  export let alt = 'Vercel.com Logo'
```

Comment:
Does this need to be a prop? When would it ever be anything other than `Vercel.com logo`?